src/scripts/settings/userSettings.js

```diff
@@ -429,6 +429,25 @@ export class UserSettings {
         }
     }
 
+    /**
+     * Get or set max days for next up list.
+     * @param {number|undefined} val - Max days for next up.
+     * @return {number} Max days for a show to stay in next up without being watched.
+     */
+    maxDaysForNextUp(val) {
+        if (val !== undefined) {
+            return this.set('maxDaysForNextUp', parseInt(val, 10), false);
+        }
+
+        const maxDaysForNextUp = parseInt(this.get('maxDaysForNextUp', false), 10);
+        if (maxDaysForNextUp === 0) {
+            // Explicitly return 0 to avoid returning 100 because 0 is falsy.
+            return 0;
+        } else {
+            return maxDaysForNextUp || 100;
```

Comment:
I think 100 may be too low of a default for a field that was unlimited previously. Especially if you consider that the fallback behavior is to display the first episode of every show. This was the issue I was having in my testing. Perhaps showing one year by default would be better. :man_shrugging:

Reply:
Ya I think a year is better so changed.